### PR TITLE
Update to v0.1.4.

### PR DESCRIPTION
**Pull request by barakwei**
@Bmantl don't merge this, take a look on how they now handle the application, and LMK if you want to change our fork to remove the UIApplication parameter you added.

### CODE REVIEW

**Comment by barakwei** [comment]
@Bmantl yalla

File: Alooma-iOS/Alooma.m [hunk 1]
```diff
@@ -116,9 +121,17 @@ - (instancetype)initWithToken:(NSString *)apiToken serverURL:(NSString *)url
         self.timedEvents = [NSMutableDictionary dictionary];
         _application = application;
 
-        if (self.application) {
-          [self setUpListeners];
+        if (![Alooma isAppExtension]) {
+            if ([[UIApplication class] respondsToSelector:@selector(sharedApplication)]) {
+                _application = [[UIApplication class] performSelector:@selector(sharedApplication)];
+            }
+#if !defined(ALOOMA_APP_EXTENSION)
```
Comment (Bmantl) [inline Alooma-iOS/Alooma.m:118]: Should this be removed? same for the other one in this file.

And can we remove our addition of the nullable `application`?

**Review by Bmantl — APPROVED** [review]
One question

File: Alooma-iOS/Alooma.m (hunk 1 above)
Comment (barakwei) [reply Alooma-iOS/Alooma.m:118]: Keeping this as we don't want to diverge too much from upstream now that the issue is fixed.

I changed the PR to keep the diff from upstream to a minimum. Note that they added a nullable `UIApplication` property.

I suggest to manually look at the diff from `upstream/master`.

**Review by Bmantl — APPROVED** [review]
LGTM

**Comment by barakwei** [comment]
Tested E2E

**Pull request merged**